mailgun/resource_mailgun_domain.go

```diff
@@ -482,3 +480,26 @@ func ListCredentials(domain, apiKey string) ([]mailgun.Credential, error) {
 	}
 	return result, nil
 }
+
+func ImportStatePassthroughDomain(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	if _, ok := d.GetOk("dkim_key_size"); !ok {
+		d.Set("dkim_key_size", 1024)
+	}
+
+	if _, ok := d.GetOk("force_dkim_authority"); !ok {
+		d.Set("force_dkim_authority", false)
+	}
+	return []*schema.ResourceData{d}, nil
+}
+
+func getIps(ctx context.Context,mg *mailgun.MailgunImpl) ([]mailgun.IPAddress, error){
+	start := time.Now()
+	t := time.Now()
+	ipAddress, err :=mg.ListDomainIPS(ctx)
```

Comment:
can you add a space between `=` and `mg` please?

Reply:
I'm doing it